smartcar-auth/src/main/java/com/smartcar/sdk/SmartcarAuth.java

```diff
@@ -170,6 +170,18 @@ public AuthUrlBuilder setSingleSelectVin(String vin) {
             return this;
         }
 
+        /**
+         * Enable early access features.
+         *
+         * @param flags List of early access features to enable. The naming of the flags is colon
+         * separated for any sub-options the flag may have (ex: country:DE).
```

Comment:
```suggestion
         * @param flags List of feature flags that your application has early access to.
```

I would say to leave out the whole sub-options bit and just make it super-generic in the description. I pulled that copy from the API reference.

Reply:
Sounds good! will update the documentation for the other SDKs as well.